external/akka/src/main/scala/org/apache/spark/streaming/akka/ActorReceiver.scala

```diff
@@ -202,12 +294,11 @@ private[akka] case class ByteBufferData(bytes: ByteBuffer) extends ActorReceiver
  * }}}
  */
 private[akka] class ActorReceiverSupervisor[T: ClassTag](
-    actorSystemCreator: () => ActorSystem,
-    props: Props,
-    name: String,
-    storageLevel: StorageLevel,
-    receiverSupervisorStrategy: SupervisorStrategy
-  ) extends Receiver[T](storageLevel) with Logging {
+   actorSystemCreator: () => ActorSystem,
```

Comment:
If I am not wrong, arguments are indented 4 spaces, are you sure this adheres to the style guide ? [Style guide](https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Indentation)